web/html/src/utils/functions.ts

```diff
@@ -43,52 +42,65 @@ function dateWithTimezone(dateString: string): Date {
 
 // it does the opposite of dateWithTimezone: transforms its result on the original date
 function dateWithoutTimezone(dateStringToTransform: string, originalDateString: string): Date {
-  const offsetNum = originalDateString[originalDateString.length - 1].toUpperCase() === "Z"
-    ? 0
-    : parseInt(originalDateString.substring(originalDateString.length - 6).replace(':', ''), 10);
-  const serverOffset = Math.trunc(offsetNum / 100) * 60 + offsetNum % 100;
-  const dateToTransform = new Date(dateStringToTransform);
-  const clientOffset = -dateToTransform.getTimezoneOffset();
-
-  const final = new Date(dateToTransform.getTime() - (serverOffset - clientOffset) * 60000);
-  return final;
+    const offsetNum =
+        originalDateString[originalDateString.length - 1].toUpperCase() === "Z"
+            ? 0
+            : parseInt(originalDateString.substring(originalDateString.length - 6).replace(":", ""), 10);
+    const serverOffset = Math.trunc(offsetNum / 100) * 60 + (offsetNum % 100);
+    const dateToTransform = new Date(dateStringToTransform);
+    const clientOffset = -dateToTransform.getTimezoneOffset();
+
+    const final = new Date(dateToTransform.getTime() - (serverOffset - clientOffset) * 60000);
+    return final;
 }
 
 function LocalDateTime(date: Date): string {
-    const padTo = (v) => {
+    const padTo = v => {
         v = v.toString();
-        if(v.length >= 2) return v;
+        if (v.length >= 2) return v;
         else return padTo("0" + v);
-    }
+    };
     const year = date.getFullYear();
     const month = date.getMonth();
     const days = date.getDate();
     const hours = date.getHours();
     const minutes = date.getMinutes();
     const seconds = date.getSeconds();
-    return "" + year + "-" + padTo(month + 1) + "-" + padTo(days) +
-           "T" + padTo(hours) + ":" + padTo(minutes) + ":" + padTo(seconds);
+    return (
+        "" +
+        year +
+        "-" +
+        padTo(month + 1) +
+        "-" +
+        padTo(days) +
+        "T" +
+        padTo(hours) +
+        ":" +
+        padTo(minutes) +
+        ":" +
+        padTo(seconds)
+    );
 }
 
-function sortById(aRaw: Object, bRaw: Object): number {
-  const aId = aRaw["id"];
-  const bId = bRaw["id"];
-  return aId > bId ? 1 : (aId < bId ? -1 : 0);
+function sortById(aRaw: any, bRaw: any): number {
+    const aId = aRaw["id"];
+    const bId = bRaw["id"];
+    return aId > bId ? 1 : aId < bId ? -1 : 0;
```

Comment:
Also here, we should have 2 spaces instead.

I don't think it's necessary to enforce every file to have 2 spaces, but since this one is already mixed, I think it's better to convert it 4 to 2 rather than 2 to 4.

Reply:
Thanks for pointing these out. I originally planned to address all of this in https://github.com/SUSE/spacewalk/issues/13437, but there's no reason to not already include initial work in the automated migration script. 👍